.github/workflows/push_production.yaml

```diff
@@ -0,0 +1,31 @@
+# this build is triggered when a new pre-release has been created
+# it creates a new docker build image based on the tag associated
+
+name: push_production
+
+on:
+  release:
+    types:
+      - 'prereleased'
```

Comment:
Does this mean that we will always need to go through the pre-release process first? With [the API](https://github.com/WordPress/openverse-api/blob/main/.github/workflows/ci_cd.yml#L11) we are running the build every time a release is published.

Reply:
Thank you, during the development I used `prereleased` event, forgot to update it.

Reply:
Yes @zackkrida, the name of the event should be `released`. Done in the last commit

Reply:
hm, I'm not seeing the commit you mentioned, may need to push.